librosa/filters.py

```diff
@@ -449,7 +449,7 @@ def constant_q(sr, fmin=None, n_bins=84, bins_per_octave=12, tuning=0.0,
     for i in np.arange(n_bins, dtype=float):
 
         freq = fmin * 2.0**(i / bins_per_octave)
-        if freq * (1 + 1.0 / Q) > sr / 2.0:
+        if freq * (1 + 0.725 / Q) > sr / 2.0: # 0.725 for Hann window
```

Comment:
> I don't think it would be worth computing it from scratch every time.

The only reason to do that would be for folks using home-made window functions.  I don't really expect this, but i'd rather not rule it out unless we have to.


> What are the arguments in favor of supporting many different windows?

Because we can/could.

> Do you know of anyone using something besides Hann?

Not off-hand.  I'm not entirely opposed to restricting it to be Hann-only, if it's really important, but I'd rather it be as flexible as we can get away with.


> I like the dictionary idea.

This would be pretty easy to pull off if we can come up with a list of cutoffs for each of the scipy windows.